sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -130,105 +132,126 @@ def analyze_text():
         if e.error is not None:
             print("Error code: {}".format(e.error.code))
             print("Error message: {}".format(e.error.message))
-            return
+            raise
         print(e)
         raise
 
     if response.hate_result is not None:
         print("Hate severity: {}".format(response.hate_result.severity))
     if response.self_harm_result is not None:
         print("SelfHarm severity: {}".format(response.self_harm_result.severity))
-
-
-if __name__ == "__main__":
-    analyze_text()
 ```
 
 <!-- END SNIPPET -->
 
-### Analyze image
-
-<!-- SNIPPET:sample_analyze_image.analyze_image -->
+#### Analyze text with blocklists
+<!-- SNIPPET:sample_manage_blocklist.analyze_text_with_blocklists -->
 
 ```python
-import os
-from azure.ai.contentsafety import ContentSafetyClient
-from azure.core.credentials import AzureKeyCredential
-from azure.core.exceptions import HttpResponseError
-from azure.ai.contentsafety.models import AnalyzeImageOptions, ImageData
 
+    import os
+    from azure.ai.contentsafety import ContentSafetyClient
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.contentsafety.models import AnalyzeTextOptions
+    from azure.core.exceptions import HttpResponseError
 
-def analyze_image():
     key = os.environ["CONTENT_SAFETY_KEY"]
     endpoint = os.environ["CONTENT_SAFETY_ENDPOINT"]
-    image_path = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "./sample_data/image.jpg"))
 
     # Create an Content Safety client
     client = ContentSafetyClient(endpoint, AzureKeyCredential(key))
 
-    # Build request
-    with open(image_path, "rb") as file:
-        request = AnalyzeImageOptions(image=ImageData(content=file.read()))
+    blocklist_name = "TestBlocklist"
+    input_text = "I h*te you and I want to k*ll you."
 
-    # Analyze image
     try:
-        response = client.analyze_image(request)
+        # After you edit your blocklist, it usually takes effect in 5 minutes, please wait some time before analyzing with blocklist after editing.
+        analysis_result = client.analyze_text(AnalyzeTextOptions(text=input_text, blocklist_names=[blocklist_name], break_by_blocklists=False))
+        if analysis_result is not None and analysis_result.blocklists_match_results is not None:
```

Comment:
Usually, in Python we don't do `is not None`, the fact that the object exists is enough:
```suggestion
        if analysis_result and analysis_result.blocklists_match_results:
```

Reply:
Removed "is not None"